src/lsptoolshost/onAutoInsert.ts

```diff
@@ -40,7 +40,15 @@ export function registerOnAutoInsert(languageServer: RoslynLanguageServer) {
 
             source.cancel();
             source = new vscode.CancellationTokenSource();
-            await applyAutoInsertEdit(e, changeTrimmed, languageServer, source.token);
+            try {
+                await applyAutoInsertEdit(e, changeTrimmed, languageServer, source.token);
+            } catch (e) {
```

Comment:
We already had an 'e' in the parent scope -- this might not be bad but could get confusing.